lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -2251,11 +2251,16 @@ $attribute_list::5
 ]]>
       </xs:documentation>
     </xs:annotation>
-    <xs:attribute name="value" type="Bytes" use="optional">
+    <xs:attribute name="size" type="Bytes" use="optional">
       <xs:annotation>
         <xs:documentation xml:lang="en">Desired size of the output (in bytes), can be suffixed by ``(k|M|G|T|P|E)i?``</xs:documentation>
       </xs:annotation>
     </xs:attribute>
+    <xs:attribute name="value" type="Bytes" use="optional">
+      <xs:annotation>
+        <xs:documentation xml:lang="en">An outdated synonym for `size`</xs:documentation>
```

Comment:
```suggestion
        <xs:documentation xml:lang="en">An outdated alias for `size`</xs:documentation>
```

Reply:
Hah, that was the word I was looking for.